redaxo/src/core/pages/system.ext-logs.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @package redaxo5
+ */
+
+$func = rex_request('func', 'string');
+$logFile = ini_get('error_log');
+/*
+$logFile = rex_request('file', 'string');
+$availableLogs = [ini_get('error_log')];
+
+if (!in_array($logFile, $availableLogs)) {
+    unset($logFile);
+}
+*/
+
+$error = '';
+$success = '';
+
+if ($func == 'delLog') {
+    if (rex_file::delete($logFile)) {
+        $success = rex_i18n::msg('extlog_deleted', $logFile);
+    } else {
+        $error = rex_i18n::msg('extlog_delete_error', $logFile);
+    }
+}
+$message = '';
+$content = '';
+
+if ($success != '') {
+    $message .= rex_view::success($success);
+}
+
+if ($error != '') {
+    $message .= rex_view::error($error);
+}
+
+if (!isset($logFile)) {
+    return;
+}
+
+$content .= '
+            <table class="table table-hover">
+                <tbody>';
+
+$file = new SplFileObject($logFile, 'r');
+$file->seek(PHP_INT_MAX);
+$last_line = $file->key();
+
+$limit = 10;
+foreach (new LimitIterator($file, $last_line - $limit, $last_line) as $logLine) {
```

Comment:
`OutOfRangeException`
`Parameter offset must be >= 0`
kommt wenn im errorlog <10 Zeilen drin stehen. Ansonsten hier einfach ein `max(0, $last_line - $limit)` machen.

Reply:
Danke, fixed